src/components/wishlist/WishListInfo.jsx

```diff
@@ -1,11 +1,22 @@
+import { useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
 
 function WishListInfo({ list }) {
+  const navigate = useNavigate();
+
+  const moveToWishDetail = (id) => {
+    navigate(`/wish/${id}`);
+  };
+
   return (
     <StyledWishListInfo>
       <StyledCategoryInfo>
         {list.map(({ id, image, title }) => (
-          <li key={id}>
+          <li
+            key={id}
+            onClick={() => {
+              moveToWishDetail(title);
+            }}>
```

Comment:
```suggestion
          <li
            key={id}
            onClick={() => {
              navigate(`/wish/${title}`);
            }}>
```
위에 있는 7~9번째 라인 지우고 여기에 바로 쓸 수 있어!